simpleanidb/__init__.py

```diff
@@ -54,7 +79,7 @@ def search(self, term):
                 if term in title.text.lower():
                     anime_ids.append((int(anime.get("aid")), anime))
                     break
-        return [Anime(self, aid, False, xml_node) for aid, xml_node in anime_ids]
+        return [Anime(self, aid, autoload, xml_node) for aid, xml_node in anime_ids]
```

Comment:
I'm starting the think why you've set this to false. As i was testing it, and when enabled you'l get banned in no time.